e2e/schematics/node.test.ts

```diff
@@ -120,4 +120,92 @@ describe('Node Applications', () => {
     },
     30000
   );
+
+  it(
+    'should be able to generate a nestjs application',
+    async done => {
+      runCLI('generate node-app node-app3 --framework nestjs');
+      copyMissingPackages();
+
+      updateFile(
+        'apps/node-app3/src/app/test.spec.ts',
+        `
+          describe('test', () => {
+            it('should work', () => {
+              expect(true).toEqual(true);
+            })
+          })
+        `
+      );
+      // const jestResult = await runCLIAsync('test node-app3');
+      // expect(jestResult.stderr).toContain('Test Suites: 1 passed, 1 total');
+
+      function getData() {
+        return new Promise(resolve => {
+          http.get('http://localhost:3333', res => {
+            expect(res.statusCode).toEqual(200);
+            let data = '';
+            res.on('data', chunk => {
+              data += chunk;
+            });
+            res.once('end', () => {
+              resolve(data);
+            });
+          });
+        });
+      }
+
+      await runCLIAsync('build node-app3');
+      expect(exists('./tmp/proj/dist/apps/node-app3/main.js')).toBeTruthy();
+      const server = fork(
+        path.join(
+          __dirname,
+          '../../../tmp/proj',
+          `./dist/apps/node-app3/main.js`
+        ),
+        [],
+        {
+          cwd: './tmp/proj',
+          silent: true
+        }
+      );
+      expect(server).toBeTruthy();
+
+      await new Promise(resolve => {
+        server.stdout.once('data', async data => {
```

Comment:
This only works because I disabled nest's default Logger.

With the default logger enabled, starting the server logs multiple lines, where only the last will be `Listening at http://localhost:3333`. How can I change this to have a short timeout and then get the last printed line?